cmodelres/CMakeLists.txt

```diff
@@ -129,3 +154,28 @@ if(SEA_EXE)
         COMMAND_EXPAND_LISTS
     )
 endif()
+
+set(CMAKE_C_COMPILER clang)
+set(LLVM_LINK llvm-link)
+set(KLEE_DRIVER_BC "klee_driver.bc")
+set(KLEE_SRC "/home/klee/klee_src/include")
+
+# target for KLEE
+if(KLEE_EXE)
```

Comment:
Maybe this should use KLEE_PATH and we should introduce a nested if for the symbex target that checks if KLEE_EXE is set.

Reply:
Yeah restructuring a bit could make sense, but given my knowledge of klee, it lgtm. My schedule's pretty packed this evening, but I'll try to propose an update, time permitting.

Reply:
Great! I think @huxinwen1994 addressed this. But if you have any other improvements please go ahead. :)